examples/Kaskada Demo for Event Processing and Time-centric Calculations.ipynb

```diff
@@ -0,0 +1,559 @@
+    "* the total number of purchases the user has made\n",
+    "* the total revenue from the user\n",
+    "* the number of purchases made by the user today\n",
+    "* the total revenue from the user today\n",
+    "* the number of events the user has generated in the past hour\n",
+    "\n",
+    "Because the calculations needed here are a mix of hourly, daily, and over all of\n",
+    "history, more than one type of event aggregation needs to happen. Table-centric\n",
+    "tools like those based on SQL would require multiple JOINs and window functions,\n",
+    "which would be spread over multiple queries or CTEs. \n",
+    "\n",
+    "Kaskada was designed for these types of time-centric calculations, so we can do\n",
+    "each of the calculations in the list in one line:\n",
+    "\n",
+    "```\n",
+    "event_count_total: DemoEvents | count(),\n",
+    "purchases_total_count: DemoEvents | when(DemoEvents.event_name == 'purchase') | count(),\n",
+    "revenue_total: DemoEvents.revenue | sum(),\n",
+    "purchases_daily: DemoEvents | when(DemoEvents.event_name == 'purchase') | count(window=since(daily())),\n",
+    "revenue_daily: DemoEvents.revenue | sum(window=since(daily())),\n",
+    "event_count_hourly: DemoEvents | count(window=since(hourly())),\n",
+    "\n",
+    "```\n",
+    "\n",
+    "Of course, a few more lines of code is needed to put these calculations to work,\n",
+    "but these six lines are all that is needed to specify the calculations\n",
+    "themselves. Each line may specify:\n",
+    "* the name of a calculation (e.g. `event_count_total`)\n",
+    "* the input data to start with (e.g. `DemoEvents`)\n",
+    "* selecting event fields (e.g. `DemoEvents.revenue`)\n",
+    "* function calls (e.g. `count()`)\n",
+    "* event filtering (e.g. `when(DemoEvents.event_name == 'purchase')`)\n",
+    "* time windows to calculate over (e.g. `window=since(daily())`)\n",
+    "\n",
+    "...with consecutive steps separated by a familiar pipe (`|`) notation.\n",
+    "\n",
+    "Because Kaskada was built for time-centric calculations on event-based data, a\n",
+    "calculation we might describe as \"total number of purchase events for the user\"\n",
+    "can be defined in Kaskada in roughly the same number of terms as the verbal\n",
+    "description itself.\n",
+    "\n",
+    "Continue through the demo below to find out how it works.\n",
+    "\n",
+    "See [the Kaskada\n",
+    "documentation](https://kaskada-ai.github.io/docs-site/kaskada/main/getting-started/installation.html)\n",
+    "for lots more information."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "e06233eb",
+   "metadata": {},
+   "source": [
+    "## Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bbe86c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install --upgrade pip\n",
+    "%pip install kaskada==0.1.1a7\n",
```

Comment:
I think we just released a non-alpha version so we might want to use 

```
pip install kaskada
```

Reply:
makes sense!